customer_satisfaction/streamlit_app.py

```diff
@@ -0,0 +1,96 @@
+    | Models        | Description   | 
+    | ------------- | -     | 
+    | Payment Sequential | Customer may pay an order with more than one payment method. If he does so, a sequence will be created to accommodate all payments. | 
+    | Payment Installments   | Number of installments chosen by the customer. |  
+    | Payment Value |       Total amount paid by the customer. | 
+    | Price |       Price of the product. |
+    | Frieght Value |    Frieght value of the product.  | 
+    | Product Name length |    Length of the product name. |
+    | Product Description length |    Length of the product description. |
+    | Product photos Quantity |    Number of product published photos |
+    | Product weight measured in grams |    Weight of the product measured in grams. | 
+    | Product length (CMs) |    Length of the product measured in centimeters. |
+    | Product height (CMs) |    Height of the product measured in centimeters. |
+    | Product width (CMs) |    Width of the product measured in centimeters. |
+    """
+    )
+    payment_sequential = st.sidebar.slider("Payment Sequential") 
+    payment_installments = st.sidebar.slider("Payment Installments")  
+    payment_value = st.number_input("Payment Value") 
+    price = st.number_input("Price")  
+    freight_value = st.number_input("freight_value") 
+    product_name_lenght = st.number_input("Product name length")  
+    product_description_lenght = st.number_input("Product Description length") 
+    product_photos_qty = st.number_input("Product photos Quantity ") 
+    product_weight_g = st.number_input("Product weight measured in grams") 
+    product_length_cm = st.number_input("Product length (CMs)") 
+    product_height_cm = st.number_input("Product height (CMs)")  
+    product_width_cm = st.number_input("Product width (CMs)")
+
+    result = "" 
+    if st.button("Predict"):
+        with open('saved_model/model.pkl', 'rb') as handle:  
+            model = pickle.load(handle) 
+        result = predict(model, payment_sequential,
+                        payment_installments,
+                        payment_value,	
+                        price,	
+                        freight_value,	
+                        product_name_lenght	,
+                        product_description_lenght,	
+                        product_photos_qty,	
+                        product_weight_g,	
+                        product_length_cm,
+                        product_height_cm,
+                        product_width_cm) 
+        st.success('Your Customer Satisfactory rate(range between 0 - 5) with given product details is :-{}'.format(result))
+    if st.button("Results"): 
+        st.write("We have experimented with 2 ensemble and tree based models and compared the performance of each model. The results are as follows:")
```

Comment:
```suggestion
        st.write("We have experimented with two ensemble and tree based models and compared the performance of each model. The results are as follows:")
```

Generally it's best for numbers below 11 to write them out fully.